src/openrct2/actions/WallPlaceAction.h

```diff
@@ -15,19 +15,13 @@
 #include "../world/Scenery.h"
 #include "GameAction.h"
 
-class WallPlaceActionResult final : public GameActions::Result
+struct WallPlaceActionResult
 {
-public:
-    WallPlaceActionResult();
-    WallPlaceActionResult(GameActions::Status err);
-    WallPlaceActionResult(GameActions::Status err, rct_string_id msg);
-    WallPlaceActionResult(GameActions::Status error, rct_string_id msg, uint8_t* args);
-
-    TileElement* tileElement = nullptr;
-    BannerIndex bannerId = BANNER_INDEX_NULL;
+    TileElement* Element = nullptr;
```

Comment:
Same as the other action I don't think we should be passing about a pointer. Instead pass back the height.